### ISSUE TITLE
print_mem_dump fails when one symbol doesn't exist on system

### ISSUE THREAD

**Issue by NDStrahilevitz** [aquasecurity/tracee#3383]
The `print_mem_dump` event can dump the function of many different functions loaded in the kernel, as many as are loaded in its "argument" parameters.
However, its handler code currently hard-fails when any symbol query fails. This means the event will not be generated, despite still being useful for other symbols.
```
tracee -e print_mem_dump.args.symbol_name=compat_filldir64
TIME             UID    COMM             PID     TID     RET              EVENT                     ARGS
{"level":"warn","ts":1691929975.2542894,"msg":"print_mem_dump returned an error","error":"ebpf.(*Tracee).triggerMemDump: symbol not found: system___arm64_sys_compat_filldir64"}
^C
End of events stream
Stats: {EventCount:{value:0} EventsFiltered:{value:0} NetCapCount:{value:0} BPFLogsCount:{value:0} ErrorCount:{value:0} LostEvCount:{value:0} LostWrCount:{value:0} LostNtCapCount:{value:0} LostBPFLogsCount:{value:0}}
```

**Comment by rafaeldtinoco** [comment]
Fix for this issue was reverted at https://github.com/aquasecurity/tracee/pull/3436. I'll keep this issue opened because the proposed fix for this brakes E2E tests and needs further investigation.

**Comment by rafaeldtinoco** [comment]
Actually, after the fix was reverted, I'm not sure if the issue really exists. The following reproducer:

```
rafaeldtinoco@rugged:tracee$ sudo ./dist/tracee -e print_mem_dump.args.symbol_name=do_sys_open,bleh,proc_sys_open
TIME             UID    COMM             PID     TID     RET              EVENT                     ARGS
{"level":"warn","ts":1693947328.466371,"msg":"print_mem_dump returned an error","error":"ebpf.(*Tracee).triggerMemDump: symbol not found: system___arm64_sys_bleh"}
21:00:00:000000  0                       0       0       0                print_mem_dump            bytes: [243 15 30 250 15 31 68 0 0 65 84 72 199 192 200 155 210 175 73 137 244 85 72 137 253 83 72 139 95 216 72 199 199 96 22 116 176 72 133 219 72 15 68 216 232 127 50 141 0 72 131 123 24 0 117 106 131 67 8 1 72 199 199 96 22 116 176 232 40 51 141 0 72 139 69 224 72 129 251 0 240 255 255 119 96 72 139 64 40 72 133 192 116 11 72 99 0 73 137 132 36 200 0 0 0 72 199 199 96 22 116 176 232 59 50 141 0 131 107 8 1 116 23 72 199 199 96], address: 0xffffffffae2aeaa0, length: 127, caller_context_id: 1, arch: x86_64, symbol_name: proc_sys_open, symbol_owner: system
21:00:00:000000  0                       0       0       0                print_mem_dump            bytes: [102 15 31 0 15 31 68 0 0 72 131 236 32 101 72 139 4 37 40 0 0 0 72 137 68 36 24 137 208 247 194 0 0 32 0 116 58 37 0 0 43 0 49 201 72 137 226 72 137 4 36 72 137 76 36 8 72 199 68 36 16 0 0 0 0 232 154 244 255 255 72 139 84 36 24 101 72 43 20 37 40 0 0 0 117 37 72 131 196 32 195 204 204 204 204 129 225 255 15 0 0 37 195 255 127 0 129 226 64 0 64 0 186 0 0 0 0 72 15 68 202 235 177 232 64 147 151], address: 0xffffffffae1f1a00, length: 127, caller_context_id: 2, arch: x86_64, symbol_name: do_sys_open, symbol_owner: system

End of events stream
Stats: {EventCount:{value:2} EventsFiltered:{value:0} NetCapCount:{value:0} BPFLogsCount:{value:0} ErrorCount:{value:0} LostEvCount:{value:0} LostWrCount:{value:0} LostNtCapCount:{value:0} LostBPFLogsCount:{value:0}}
```

seems to be working ok.

**Comment by rafaeldtinoco** [comment]
Although there is a bug: it says "triggerMemDump: symbol not found: system___arm64_sys_bleh"}" when the argument was just "bleh". I wonder why.

**Comment by geyslan** [comment]
#3497 which solved this `print_mem_dump` issue has been reverted as it's triggering a `symbols_loader` dormant issue.

When the "symbols_loader" issue is solved, I'll re-raise #3497.

**Comment by geyslan** [comment]
@roikol do you mind taking a look at this?
